src/Tes.ApiClients/Models/Terra/DrsResolveApiResponse.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Tes.ApiClients.Models.Terra
+{
+
+    public class DrsResolveApiResponse
+    {
+        [JsonPropertyName("contentType")]
+        public string ContentType { get; set; }
+
+        [JsonPropertyName("size")]
+        public long Size { get; set; }
+
+        [JsonPropertyName("timeCreated")]
+        public DateTime TimeCreated { get; set; }
```

Comment:
Do we assume it's UTC?  If so, maybe name it TimeCreatedUtc. 
 Alternatively use the `DateTimeOffset` type